<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="4">
         <widget class="QPushButton" name="pushButtonSampDown">
          <property name="text">
           <string>Samples Down</string>
          </property>
         </widget>
        </item>
        <item row="1" column="7">
         <widget class="QSpinBox" name="spinBoxLFPpercent">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>300</number>
          </property>
          <property name="singleStep">
           <number>5</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="QLabel" name="labelLFP">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>LFP scale %</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelStart">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="pushButtonAdvance">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Advance</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditStart">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEditRecords">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>7680</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButtonGo">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Go</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelRecords">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Samples</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QLabel" name="labelFolder">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="7">
         <widget class="QPushButton" name="pushButtonSet">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
        <item row="1" column="8">
         <widget class="QPushButton" name="pushButtonSpikes">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Spikes</string>
          </property>
         </widget>
        </item>
        <item row="3" column="8">
         <widget class="QPushButton" name="pushButtonSave">
          <property name="text">
           <string>Save Image</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="pushButtonBack">
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="pushButtonSampUp">
          <property name="text">
           <string>Samples Up</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>877</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuChannels">
    <property name="title">
     <string>Channels</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTraces">
    <property name="title">
     <string>Traces</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionShowBoxes"/>
    <addaction name="actionShow_spikes"/>
    <addaction name="actionUse_wall_time"/>
   </widget>
   <widget class="QMenu" name="menuRefs">
    <property name="title">
     <string>Montages</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSleepstaging">
    <property name="title">
     <string>Sleepstaging</string>
    </property>
    <addaction name="action_W"/>
    <addaction name="action_N1"/>
    <addaction name="action_N2"/>
    <addaction name="action_N3"/>
    <addaction name="action_R"/>
    <addaction name="action_Save_to_file"/>
   </widget>
   <addaction name="menuChannels"/>
   <addaction name="menuRefs"/>
   <addaction name="menuTraces"/>
   <addaction name="menuOptions"/>
   <addaction name="menuSleepstaging"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLFP">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Raw</string>
   </property>
  </action>
  <action name="actionShowBoxes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show detection boxes</string>
   </property>
  </action>
  <action name="actionShow_spikes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show spikes</string>
   </property>
  </action>
  <action name="actionSimple">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Simple</string>
   </property>
  </action>
  <action name="actionAdvance">
   <property name="text">
    <string>Advance</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="text">
    <string>Back</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionUse_wall_time">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use wall time</string>
   </property>
  </action>
  <action name="action_R">
   <property name="text">
    <string>&amp;REM</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="action_N1">
   <property name="text">
    <string>N&amp;1</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="action_N2">
   <property name="text">
    <string>N&amp;2</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="action_N3">
   <property name="text">
    <string>N&amp;3</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="action_W">
   <property name="text">
    <string>&amp;W</string>
   </property>
   <property name="shortcut">
    <string>W</string>
   </property>
  </action>
  <action name="actionSampUp">
   <property name="text">
    <string>actionSampUp</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionSampDown">
   <property name="text">
    <string>actionSampDown</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="action_Save_to_file">
   <property name="text">
    <string>&amp;Save to file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
